### PR TITLE
update to actual file name

### PR DESCRIPTION
**Pull request by matkoniecz**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ If you have something awesome to contribute to this list, this is how you do it.
 You'll need a [GitHub account](https://github.com/join)!
 
 1. Access the list's GitHub page: https://github.com/maplibre/awesome-maplibre
-2. Click on the `readme.md` file: ![Step 2 Click on Readme.md](https://cloud.githubusercontent.com/assets/170270/9402920/53a7e3ea-480c-11e5-9d81-aecf64be55eb.png)
+2. Click on the `README.md` file: ![Step 2 Click on Readme.md](https://cloud.githubusercontent.com/assets/170270/9402920/53a7e3ea-480c-11e5-9d81-aecf64be55eb.png)
```
[the remark this answers is not in the document]
Comment (matkoniecz) [reply contributing.md:10]: I am unlikely to have time for that.

[the remark this answers is not in the document]
Comment (JannikGM) [reply contributing.md:10]: Just to clarify: I meant the stuff in the square brackets. We don't necessarily have to update the image (which, indeed, complicates the PR too much).

[the remark this answers is not in the document]
Comment (HarelM) [reply contributing.md:10]: I changed readme to capital, maybe only partially...

[the remark this answers is not in the document]
Comment (matkoniecz) [reply contributing.md:10]: "Just to clarify: I meant the stuff in the square brackets" oh, I missed it and misunderstood you. Sorry.

Is this part even shown anywhere? It appears to be not used even as a tooltip.

**Pull request closed without merging**